package.json

```diff
@@ -33,6 +33,7 @@
     "mocha-jshint": "^2.2.3"
   },
   "dependencies": {
+    "angular-moment": "^0.10.3",
```

Comment:
This is backend package.json. Is angular-moment really used in the backend code?